examples/payments.php

```diff
@@ -25,9 +26,28 @@
     'paymentType',
 ];
 
-// Call authenticate first to get an access token
-$response = $client->authenticate()
-    ->get(\Dash\Client::buildIndexRequestUri('payments', $filters, $includes));
+// increase the page size from 15 (default) to 25
+$page = new \Dash\PageObject(1, 25);
+
+// add any custom query parameters such as telling filters to apply to included relationships
+$custom = [
+    'filterRelations' => true,
+];
+
+$response = $client->get(\Dash\Client::buildIndexRequestUri('payments', $filters, $includes, null, $page, $custom));
 
 // decode the json data to associative array
-$data = json_decode($response->getBody()->getContents(), true);
\ No newline at end of file
+$responseArray = json_decode($response->getBody()->getContents(), true);
+$data = $responseArray['data'];
+
+// simplified method of retrieving all pages
+// NOTE: API endpoints are rate-limited (results in 429 status code) so final solutions need to be able to deal with that
+if ($responseArray['meta']['page']['current-page'] !== $responseArray['meta']['page']['last-page']) {
```

Comment:
@Wizofgoz should we support an iterator we allow to go through the pages of a response?